docs/development/when_to_rebase.rst

```diff
@@ -19,13 +19,24 @@ of commits is excessive for the changes made.  The definition of 'excessive' is
 subjective, but would never apply for 5 or fewer commits even for a trivial
 change.  Reviewers should use their discretion in suggesting to combine/squash
 commits.  Be mindful of maintaining a welcoming environment and be helpful with
-advice, especially for new contributors.
+advice, especially for new contributors.  E.g., maintainers should avoid
+demanding a squash from a novice git user unfamiliar with rebasing (although
```

Comment:
I don't particularly like `demanding` here, no point in a review is demanding really. So can we tune it down a bit to say something like "maintainers can ask a squash from ...., but it is expected to offer help to the contributor or do the squash themselves by directly pushing to the PR branch"?

Reply:
good point.  Will fix